sdl_android/src/androidTest/java/com/smartdevicelink/test/rpc/requests/GetVehicleDataTests.java

```diff
@@ -162,6 +165,7 @@ public void testRpcValues () {
         assertNull(Test.NULL, msg.getEmergencyEvent());
         assertNull(Test.NULL, msg.getClusterModeStatus());
         assertNull(Test.NULL, msg.getMyKey());
+	    assertNull(Test.NULL, msg.getElectronicParkBrakeStatus());
```

Comment:
Not sure how picky we want to be with tabs/spaces but there are a lot of lines in this PR that dont line up